src/app/app.component.html

```diff
@@ -1,13 +1,9 @@
 <next-stepper
   [firstStep]="'firstStep'"
-  [firstChecked]="firstChecked"
-  [firstDisabled]="firstDisabled"
   [secondStep]="'secondStep'"
-  [secondChecked]="secondChecked"
-  [secondDisabled]="secondDisabled"
   [thirdStep]="'thirdStep'"
-  [thirdChecked]="thirdChecked"
-  [thirdDisabled]="thirdDisabled"
+
+  [checked]=""
```

Comment:
not sure that really understand the point of this input 
may be better to change the name of the variable

Reply:
checked is number of current step. It can be 0,1 or 2. If it's empty or doesn't exist it'll be 0